### ISSUE TITLE
About metadata

### ISSUE THREAD

**Issue by btspoony** [muttoni/flow-contract-browser#1]
Maybe using collectionDisplay from [NFT Catalog](https://github.com/dapperlabs/nft-catalog)  is better for metadata?

https://github.com/muttoni/flow-contract-browser/blob/main/src/lib/metadata.js

**Comment by muttoni** [comment]
Good idea! I've removed the metadata for now as we're rebuilding it from scratch.
